packages/rum-core/src/common/compress.js

```diff
@@ -289,9 +289,9 @@ export function compressError(error) {
 }
 
 export function compressMetricsets(breakdowns) {
-  return breakdowns.map(({ span, samples }) => {
-    const isSpan = span != null
-    if (isSpan) {
+  return breakdowns
+    .filter(({ span }) => span != null)
```

Comment:
now that I'm thinking, this filter shouldn't be necessary anymore, since now all breakdowns are for spans, I will check this.